ecommerce/frontend/src/components/pages/SignupPage.js

```diff
@@ -6,41 +6,40 @@ import { sleep } from '../../utils'
 import { notification, Spin } from 'antd'
 import { useState } from 'react'
 export const SignupPage = () => {
-    const [isLoading, setIsLoading] = useState(false);
+    const [isLoading, setIsLoading] = useState(false)
 
     const history = useHistory()
 
-    const onSubmit = async (values, userType) =>  {
-        if(userType === "customer") {
+    const onSubmit = async (values, userType) => {
+        if (userType === 'customer') {
             try {
-                const { data } = await api.post("/regularsignup/", values)
-                if(data.successful) {
-                    notification.success({message:"You have successfully signed up!"})
+                const { data } = await api.post('/regularsignup/', values)
+                if (data.successful) {
+                    notification.success({ message: 'You have successfully signed up!' })
                     setIsLoading(true)
                     await sleep(2000)
                     setIsLoading(false)
                     history.push({
-                        pathname: "/login"
+                        pathname: '/login',
                     })
+                } else {
+                    notification.warning({ message: data.message })
                 }
-                else {
-                    notification.warning({message: data.message})
-                }
-
             } catch (error) {
-                notification.warning({message: "There was an error with your request."})
-                
-            } 
+                notification.warning({ message: 'There was an error with your request.' })
+            }
         }
         return
     }
```

Comment:
since we already have regularLogin in AppContext, please move regular signup logic there to (regularSignup) for example